src/js/components/HashMapDisplay.js

```diff
@@ -114,7 +140,11 @@ HashMapDisplay.propTypes = {
   key: React.PropTypes.string,
   // Optional object with keys consisting of keys in `props.hash` to be
   // replaced, and with corresponding values of the replacement to be rendered.
-  renderKeys: React.PropTypes.object
+  renderKeys: React.PropTypes.object,
+  showActions: React.PropTypes.bool,
+  onEditClick: React.PropTypes.func,
+  keyPath: React.PropTypes.array,
+  emptyValue: React.PropTypes.string
 };
 
 module.exports = HashMapDisplay;
```

Comment:
I think we should remove all the edits to this file and remove the edit button on each row. I thought we weren't going to add this till we could add it in the marathon config too? This `ConfigurationHashMap` component is problematic and we should (not in this PR) find a good way to get rid of it and instead just create the list of config key/value pairs like this:
```
item.map((index, item) => {   
   const [key, value] = item;
  
    return (
        <ConfigurationMapRow key={index}>
          <ConfigurationMapLabel keepTextCase={isAttribute}>
            {key}
          </ConfigurationMapLabel>
          <ConfigurationMapValue>{value}</ConfigurationMapValue>
        </ConfigurationMapRow>
      );
});
```
Until we do this, we have no flexibility for adding things like edit buttons without crazy props and key-path joining/splitting etc. cc/ @dcos/frontend-foundation